doc/conf.py

```diff
@@ -254,3 +262,31 @@ def setup(app):
 #    html_theme = 'alabaster'
 # else:
 #    html_theme = 'nat'
+
+models_with_documentation = (
+    "models/multimeter",
+    "models/spike_detector",
+    "models/weight_recorder",
+    "nestkernel/recording_backend_ascii",
+    "nestkernel/recording_backend_memory",
+    "nestkernel/recording_backend_screen",
+    "nestkernel/recording_backend_sionlib",
+)
+
+import re
+import os
```

Comment:
These imports violate PEP8 rule "E402 module level import not at top of file".
`Import os `is furthermore duplicate and can be removed.